internal/viridian/cluster.go

```diff
@@ -65,34 +68,40 @@ func (a API) StopCluster(ctx context.Context, idOrName string) error {
 	if err != nil {
 		return err
 	}
-	ok, err := doPost[[]byte, bool](ctx, fmt.Sprintf("/cluster/%s/stop", c.ID), a.Token(), nil)
+	ok, err := WithRetry(ctx, a, func() (any, error) {
```

Comment:
Does returning `bool` instead of `any` cause any problems?

Reply:
Fixed in: https://github.com/hazelcast/hazelcast-commandline-client/pull/300/commits/d3d3b8eb6da57e1de512a87e7ebd5597a1c31c6f